src/models/spaces/organisms/organisms-space.ts

```diff
@@ -124,14 +125,20 @@ export const OrganismsSpaceModel = types
           self.organismsMice.push(organismsMouse);
         }
 
+        // keep right panel as it was
```

Comment:
I see what you mean, recreating the rows each time a mouse is created/destroyed has caused a couple of problems so far. Maybe that was a mistake to begin with, even though it previously simplified resetting.